src/materialized/src/bin/materialized/sys.rs

```diff
@@ -17,7 +17,9 @@ use std::ptr;
 use std::sync::atomic::{AtomicUsize, Ordering};
 
 use anyhow::{bail, Context};
-use log::{trace, warn};
+use log::trace;
+#[cfg(any(target_os = "macos", target_os = "linux", target_os = "ios"))]
+use log::warn;
```

Comment:
In the interest of consistency, can you push this into the `adjust_rlimits` method below, next to the `use rlimit::{Resource, Rlim}` line?

Reply:
sure